homeassistant/components/binary_sensor/tellduslive.py

```diff
@@ -9,22 +9,37 @@
 """
 import logging
 
-from homeassistant.components import tellduslive
+from homeassistant.components import binary_sensor, tellduslive
 from homeassistant.components.binary_sensor import BinarySensorDevice
 from homeassistant.components.tellduslive.entry import TelldusLiveEntity
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
 
 _LOGGER = logging.getLogger(__name__)
 
+DOMAIN = '{}_{}'.format(binary_sensor.DOMAIN, tellduslive.DOMAIN)
```

Comment:
Nope, this isn't correct either and we don't need it. Please remove this. See my comment above about the cause of the error.